blogposts/2021/release-3.34

```diff
@@ -0,0 +1,43 @@
+tags: [blog, release]
+slug: "release/3.34"
+published: false
+heroImage: https://storage.googleapis.com/sourcegraph-assets/blog/3.34/sourcegraph-3-34-release.png
+socialImage: https://storage.googleapis.com/sourcegraph-assets/blog/3.34/sourcegraph-3-34-release.png
+changelogItems:
+  - description: "Added the \"no results page\", a help page providing search tips if a search returns no results."
+    url: https://github.com/sourcegraph/sourcegraph/pull/26154
+    category: Search
+  - description: "The search indexer only polls repositories that have been marked as changed. This significantly reduces load on the Git server in installations with many repositories. If you notice index staleness, you can try disabling by setting the environment variable `SRC_SEARCH_INDEXER_EFFICIENT_POLLING_DISABLED` on `sourcegraph-frontend`."
+    url: https://github.com/sourcegraph/sourcegraph/issues/27058
+    category: Search
+  - description: "Added logging of incoming Batch Changes webhooks to help site admins debug issues with webhook delivery. By default, sites without encryption will log webhooks for three days and sites with encryption will not log webhooks."
+    url: https://docs.sourcegraph.com/admin/config/batch_changes#incoming-webhooks
+    category: Admin
+  - description: "Batch Changes now requests `workflow` scope on GitHub personal access tokens to allow writing to the `.github` directory in repositories. If you have already configured a GitHub PAT for Batch Changes, we suggest adding this scope."
+    url: https://docs.sourcegraph.com/batch_changes/how-tos/configuring_credentials
+    category: Batch Changes
+  - description: "The search UI's repositories count as well as the GraphQL API's `search().repositories` and `search().repositoriesCount` have changed semantics from the set of searchable repositories to the set of repositories with matches. This results in increased search speed, and in a future release, we'll introduce separate fields for searchable repositories."
+    url: https://github.com/sourcegraph/sourcegraph/issues/26995
+    category: Search
+  - description: "Version contexts and `repogroup` filters (deprecated in v3.33) have been removed in favor of search contexts."
+    url: https://about.sourcegraph.com/blog/introducing-search-contexts/
+    category: Search
+  - description: "Saved search notifications (deprecated in v3.31.0) have been removed in favor of code monitoring."
+    url: https://docs.sourcegraph.com/code_monitoring
+    category: Search
+---
+
+Sourcegraph 3.34 is now available! Here are some highlights from this release:
+
+## Troubleshoot your searches with the "no results" page
+Searches that return no results can often result in a confusing experience for users. When this occurs, we now display helpful tips on how to use search, examples of how to solve common query issues, and links to tutorial videos addressing common use cases.
```

Comment:
```suggestion
Searches that return no results can result in a confusing experience. When this occurs, we now display helpful tips on how to use search, examples of how to solve common query issues, and links to tutorial videos addressing common use cases.
```

Reply:
cc @benvenker just minor edits here to edit it down. The removed terms weren't adding a lot of meaning.